modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```diff
@@ -1,6 +1,9 @@
 @media (min-width: 576px) {
 	.ddm-form-builder,
 	.ddm-translation-manager {
+		.autofit-row > .autofit-col:first-child {
+			width: 100%;
+		}
 		.modal-sm {
```

Comment:
I'd expect an empty line between these. All other selectors in the file are separated by an empty line.